api_tests/institutions/views/test_institution_user_metric_list.py

```diff
@@ -195,3 +210,20 @@ def test_filter_and_pagination(self, app, url, admin, populate_more_counts):
         resp = app.get(f'{url}?filter[user_name]=Zedd', auth=admin.auth)
         assert resp.json['links']['meta']['total'] == 1
         assert resp.json['data'][0]['attributes']['user_name'] == 'Zedd'
+
+    def test_filter_and_sort(self, app, url, admin, user4, populate_counts, populate_na_department):
+        """
+        Testing for bug where sorting and filtering would throw 502.
+        :param app:
+        :param url:
+        :param admin:
+        :param populate_more_counts:
+        :return:
+        """
+        resp = app.get(f'{url}?page=1&page[size]=10&filter[department]={DEFAULT_ES_NULL_VALUE}&sort=user_name', auth=admin.auth)
```

Comment:
Since sorting by department one of the problems, could you make a second `get` to this endpoint without a filter but sorting on department?